scripts/quests/jeuno/Ducal_Hospitality.lua

```diff
@@ -0,0 +1,252 @@
+-----------------------------------
+-- Ducal Hospitality
+-- Taillegeas - !pos 31 1.996 57.971
+-- Log ID [3] - Quest ID [68]
+-----------------------------------
+require('scripts/globals/items')
+require('scripts/globals/keyitems')
+require('scripts/globals/npc_util')
+require('scripts/globals/quests')
+require('scripts/globals/status')
+require('scripts/globals/utils')
+require('scripts/globals/zone')
+require('scripts/globals/interaction/quest')
+-----------------------------------
+local quest = Quest:new(xi.quest.log_id.JEUNO, xi.quest.id.jeuno.DUCAL_HOSPITALITY)
+
+local questItemSets =
+{
+    [1] =
+    {
+        items =
+        {
+            xi.items.HANDFUL_OF_COUNTERFEIT_GIL,
+            xi.items.LOAF_OF_GOBLIN_BREAD,
+            xi.items.GOBLIN_DOLL,
+            xi.items.GOBLIN_MUSHPOT,
+            xi.items.GOBLIN_PIE,
+        },
+        textOption = 0,
+    },
+
+    [2] =
+    {
+        items =
+        {
+            xi.items.PILE_OF_CHOCOBO_BEDDING,
+            xi.items.SET_OF_KAISERIN_COSMETICS,
+            xi.items.MY_FIRST_MAGIC_KIT,
+        },
+        textOption = 1,
+    },
+
+    [3] =
+    {
+        items =
+        {
+            xi.items.PIECE_OF_EASTERN_PAPER,
+            xi.items.SHURIKEN,
+            xi.items.SILVER_OBI,
+            xi.items.TONOSAMA_RICE_BALL,
+        },
+        textOption = 4,
```

Comment:
Is this right?

Reply:
Yes -- It's sort of based on the number of items, but it handles it strangely.

Reply:
If it's the option with 5 items, it's textOption 0.  If it's 4 items, textOption 4.  3 items = textOption ... 2, I think?